src/hooks/useFormCollapse.tsx

```diff
@@ -2,7 +2,7 @@ import { type SessionMode } from '../types';
 import { useCallback, useEffect, useState } from 'react';
 
 export function useFormCollapse(sessionMode: SessionMode) {
-  const [isFormExpanded, setIsFormExpanded] = useState(true);
+  const [isFormExpanded, setIsFormExpanded] = useState<boolean | undefined>(undefined);
```

Comment:
```suggestion
  const [isFormExpanded, setIsFormExpanded] = useState<FormExpanded>();
```


(You may have to import the type `FormExpanded`)